geode-core/src/distributedTest/java/org/apache/geode/internal/tcp/CloseConnectionTest.java

```diff
@@ -40,14 +41,15 @@
 import org.apache.geode.test.dunit.rules.DistributedRule;
 
 public class CloseConnectionTest implements Serializable {
+  private static final long serialVersionUID = 3692493564204797623L;
   private VM vm0;
   private VM vm1;
 
   @Rule
   public DistributedRule distributedRule = new DistributedRule();
 
   @Rule
-  public CacheRule cacheRule = new CacheRule();
+  public CacheRule cacheRule = new CacheRule.Builder().addConfig(CONSERVE_SOCKETS, "true").build();
```

Comment:
Perhaps if this test is explicitly testing the conserver socket `true` scenario the name of the test class or some java docs should indicate that.

Reply:
The test class only contains one test case, which is `sharedSenderShouldRecoverFromClosedSocket()`. It's not immediately obvious that this implies `conserve-sockets=true` so adding some javadocs would be a good idea, I think.